examples/chip-tool/BUILD.gn

```diff
@@ -92,7 +92,7 @@ static_library("chip-tool-utils") {
     sources += [ "commands/interactive/InteractiveCommands.cpp" ]
     deps += [
       "${chip_root}/examples/common/websocket-server",
-      "${chip_root}/src/platform/logging:headers",
+      "${chip_root}/src/platform:logging-header",
```

Comment:
This flattens the "logging" into "platform". We currently have this in `src/app` and it is not good for maintainabilit - we end up getting a lot of separate libraries in one large build files, without them being related. We seem to start getting this into "core" as well as we figured out we have to fix deps. Flattening things out into "platform" seems  undesirable.